tox.ini

```diff
@@ -29,8 +29,7 @@ passenv =
     SF_PROJECT_ROOT
     cloud_provider
     SF_REGRESS_LOGS
-    ; Github Actions provided environmental variables
-    GITHUB_ACTIONS
```

Comment:
Please don't delete this

Reply:
I thought I wrongly added this line in one of my commits?

Reply:
We need it for sure, I think I added this while moving to this repo

Reply:
Sure, let's keep it